### ISSUE TITLE
Add a way to document responses

### ISSUE THREAD

**Issue by gabisurita** [Cornices/cornice.ext.swagger#4]
It would be very useful if we could do something like:

```python
class OkResponseSchema(colander.MappingSchema):
     """The request was processed sucessfully."""
     (...)

get_responses = {
    '200': MyResponseSchema()
}

@resource(path='/users/{id}')
class User(object):

    def __init__(self, request):
        self.request = request

    @view(responses=get_responses)
    def get(self):
        (...)
```

Yet I don't have a strong opinion if responses should be defined as colander schemas or just a python dict. What do you think?

Related to https://github.com/Kinto/kinto/issues/1006

**Comment by delijati** [comment]
Wow @gabisurita thats amazing. I'm currently in a other project so i won't have much time to work on cornice/ swagger.
 
@leplatrem can you give @gabisurita merge rights on this repo?

**Comment by delijati** [comment]
@leplatrem @gabisurita should we also create a ticket for that in the cornice repo?

**Comment by gabisurita** [comment]
I'm not sure about it. I think we can keep this only on `cornice.ext.swagger` right now, since I don't see any use case for it on colander itself. Do you see any problem in keeping it as a view attribute? We just should be careful with reserved words.

**Comment by leplatrem** [comment]
Maybe mention schemas in param name? `@view(response_schemas=get_responses)` ? maybe too verbose...

I think Cornice allows to add extra custom view arguments already. If it's not the case please open an issue indeed.

**Comment by gabisurita** [comment]
Closed by #23
